index.html

```diff
@@ -10,16 +10,17 @@
 
 <body>
    <section class="mobile-section">
-    <header>
+    <header id="header">
         <ul>
             <li><a href="#"><img src="Images\MyLogo.svg" alt="My Logo" width="76px" height="20px"></a></li>
             <li>
-               <nav><img src="Images\menu.svg" alt="Menu Icon" width="18px" height="12px"></nav>
+               <nav><img src="Images\menu.svg" alt="Menu Icon" width="18px" height="12px" id="mobileMenuid"></nav>
             </li>
          </ul>
+         <!-- id="mobileMenuid" onclick="open_mobile_menu()"-->
     </header>
```

Comment:
Kindly make sure that the pop-up menu stays at the top of the screen such that when a user scrolls below while the `pop-up` is open, they don't see anything other than the `pop-up` only. Currently when a user opens the `pop-up` and scrolls down they get to see all the content of the page.

Your design:
<img width="338" alt="Screenshot 2023-04-11 at 20 32 39" src="https://user-images.githubusercontent.com/107192076/231286286-982bad32-9fe9-4e92-a898-76133df1725a.png">